docs/contributing/cherry-picking.md

```diff
@@ -14,7 +14,24 @@ cherry pick.
 
 Please do not ask [core maintainers](../../OWNERS) to cherry pick your
 patch though we will be happy to review it and help merge it once the
-cherry pick PR has been submitted.
+cherry pick PR has been submitted using one of the methods below.
+
+## Method 1
+
+After your PR merges, add a comment like the following to it.
```

Comment:
You can also add the comment before the PR merges, and the bot will only create the new PR after the current PR merges

Reply:
OK, my latest update contains this information.